packages/create/src/index.ts

```diff
@@ -14,6 +14,7 @@ import {
 } from "@waku/core";
 import { PeerDiscoveryStaticPeers } from "@waku/core/lib/peer_discovery_static_list";
 import { getPredefinedBootstrapNodes } from "@waku/core/lib/predefined_bootstrap_nodes";
+import { DefaultUserAgent } from "@waku/core/lib/waku";
```

Comment:
```suggestion
import { DefaultUserAgent } from "@waku/core";
```
and in this case you need to re-export `DefaultUserAgent` in `core/src/index.ts`

OR

```suggestion
import { waku } from "@waku/core";
const { DefaultUserAgent } = waku;
```